src/app/qml/Pages/WalletPage.qml

```diff
@@ -17,6 +17,8 @@ Page {
     property string currentAssetName: assetsListView.currentItem ? assetsListView.currentItem.name : ""
     property string currentAssetColor: assetsListView.currentItem ? assetsListView.currentItem.color : ""
     property string currentAssetSymbol: assetsListView.currentItem ? assetsListView.currentItem.symbol : ""
+    property var currentBalance: assetsListView.currentItem ? assetsListView.currentItem.balance : undefined
+    property var currentUsdBalance: assetsListView.currentItem ? assetsListView.currentItem.usdBalance : undefined
```

Comment:
usdBalance is wrong, because we can configure currency, it can be euro for instance. 
My idea was that you will use dedicated class that is exposed to qml that will transform coin balance to currency balance